src/read/coff/section.rs

```diff
@@ -189,6 +190,22 @@ impl<'data, 'file, R: ReadRef<'data>> ObjectSegment<'data> for CoffSegment<'data
             .read_error("Non UTF-8 COFF section name")
             .map(Some)
     }
+
+    #[inline]
+    fn rwe_flags(&self) -> u32 {
+        let value = self.section.characteristics.get(LE);
+        let mut flag = 0;
+        if value & pe::IMAGE_SCN_MEM_READ > 0 {
```

Comment:
```suggestion
        if value & pe::IMAGE_SCN_MEM_READ != 0 {
```
When adding the methods to `SegmentFlags`, please use `!= 0` instead of `> 0`.